src/main/java/com/crud/tasks/controller/TaskController.java

```diff
@@ -4,12 +4,12 @@
 import com.crud.tasks.mapper.TaskMapper;
 import com.crud.tasks.service.DbService;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RequestMethod;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;
```

Comment:
Generalnie staramy sie nie uzywac importow z wildcartami - mozna to sobie ustawic w idei, aby nie zamieniala kilku juz importow z tego samego pakietu na wildcarty